src/targets/gpu/mlir.cpp

```diff
@@ -69,6 +69,7 @@ inline namespace MIGRAPHX_INLINE_NS {
 namespace gpu {
 
 MIGRAPHX_DECLARE_ENV_VAR(MIGRAPHX_TRACE_MLIR);
+MIGRAPHX_DECLARE_ENV_VAR(MIGRAPHX_MLIR_TUNE_EXHAUSTIVE);
```

Comment:
should we tie this to `exhaustive_tune` compile option as well ?

Reply:
I don't think so. The default of exhaustive tuning should be our "full", with full-on exhaustive being a debug feature